modules/apps/knowledge-base/knowledge-base-test/src/testIntegration/java/com/liferay/knowledge/base/web/internal/scheduler/test/CheckKBArticleSchedulerJobConfigurationTest.java

```diff
@@ -121,6 +122,40 @@ public void testExpireKBArticle() throws Exception {
 			WorkflowConstants.STATUS_EXPIRED, kbArticle.getStatus());
 	}
 
+	@Test
+	public void testNotPublishKBArticleIfKBArticleIsDraft() throws Exception {
```

Comment:
Shouldn't this be "...DoNotPublish..."? (see `testDoNotExpireKBArticleIfKBArticleIsScheduled`, for example).

Reply:
thanks!